### DISCUSSION TITLE
baidu，chatgpt，openai三个文件夹有啥区别，干嘛用；是不是调用不用开放平台的模型呢？如何切换？

### DISCUSSION THREAD

**Discussion by baichisong** [zhayujie/chatgpt-on-wechat#319] [Q&A]
bot 下面三个文件夹 baidu，chatgpt，openai三个文件夹有啥区别，干嘛用；是不是调用不用开放平台的模型呢？如何切换？

**Comment by zhayujie** [comment]
chatgpt = gpt3.5模型
openai = gpt3.0模型
baidu 目前没用

在 bridge.py 文件中修改

**Reply by baichisong** [reply]
ok
